src/Dumbo.php

```diff
@@ -29,6 +29,20 @@ class Dumbo
     /** @var array<callable> */
     private $middleware = [];
 
+    /** @var array<string, mixed> Variables stored in the context */
+    private $configs = [];
+
+    public function set(string $key, $value): void
+    {
+        $this->configs[$key] = $value;
+    }
+
+    public function getConfig(): array
```

Comment:
```suggestion
    public function get(): array: mixed
```

I would prefer we called this `get` so the API is the same application and context level.

Reply:
Isn't it confusing? Between `get` method for a request and `get` method to return configuration variable value? And you already defined the `get` method in `__call` function to register the `get` method into `route` handler.

Reply:
Ah, you make a good point. Let me think this through some more because I would prefer to be more explicit, maybe like `$app->local->get('X')` or `$app->config->get('X')`.

Maybe I'll warm to using `config`... I just think it sounds like it's a requirement using it, when it's user config/variables... and it could be any type of value stored, just just "config".